webrender/src/tiling.rs

```diff
@@ -1693,32 +1670,32 @@ pub struct StackingContextIndex(pub usize);
 
 pub struct StackingContext {
     pub pipeline_id: PipelineId,
-    local_transform: LayerToScrollTransform,
-    local_rect: LayerRect,
-    scroll_layer_id: ScrollLayerId,
-    xf_rect: Option<TransformedRect>,
-    composite_ops: CompositeOps,
-    packed_layer_index: PackedLayerIndex,
+    pub local_transform: LayerToScrollTransform,
+    pub local_rect: LayerRect,
+    pub scroll_layer_id: ScrollLayerId,
+    pub xf_rect: Option<TransformedRect>,
+    pub composite_ops: CompositeOps,
+    pub packed_layer_index: PackedLayerIndex,
 }
 
 #[derive(Debug, Copy, Clone, Eq, PartialEq, Hash, Ord, PartialOrd)]
 pub struct ScrollLayerIndex(pub usize);
 
 pub struct ScrollLayer {
-    scroll_layer_id: ScrollLayerId,
-    clip_source: ClipSource,
-    clip_cache_info: Option<MaskCacheInfo>,
-    packed_layer_index: PackedLayerIndex,
-    xf_rect: Option<TransformedRect>,
+    pub scroll_layer_id: ScrollLayerId,
```

Comment:
ok, so there is a lot of stuff that is now public
I suppose we can address the privacy concerns in follow-ups